File: silver_spring/comp jan/bacteria.java
import java.io.*;
import java.util.*;

public class bacteria {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int npatch = Integer.parseInt(br.readLine());
        int[] bacteriaLevels = new int[npatch];

        StringTokenizer st = new StringTokenizer(br.readLine());
        for (int i = 0; i < npatch; i++) {
            bacteriaLevels[i] = Integer.parseInt(st.nextToken());
        }

        int result = makeHealthy(npatch, bacteriaLevels);
        System.out.println(result);
    }

    private static int makeHealthy(int npatch, int[] bacteriaLevels) {
        int napplications = 0;

        for (int i = 0; i < npatch - 1; i++) {
            napplications += Math.abs(bacteriaLevels[i]);

            for (int j = i + 1; j < npatch; j++) {
                bacteriaLevels[j] -= (j - i + 1) * bacteriaLevels[i];
            }
        }

        napplications += Math.abs(bacteriaLevels[npatch - 1]);
        return napplications;
    }
}

File: silver_spring/comp jan/bacteria.py
def make_healthy(npatch, bacteria_levels):
    napplications = 0
    for i in range(npatch - 1):
        napplications += abs(bacteria_levels[i])
        for j in range(i + 1, npatch):
            bacteria_levels[j] -= (j - i + 1) * bacteria_levels[i]
    napplications += abs(bacteria_levels[-1])
    return napplications


if __name__ == "__main__":
    npatch = int(input())
    bacteria_levels = list(map(int, input().split()))

    result = make_healthy(npatch, bacteria_levels)
    print(result)
